migrations/20180602142104-user.js

```diff
@@ -1,45 +1,54 @@
 module.exports = {
   up: (queryInterface, DataTypes) => {
-    return queryInterface.createTable('User', {
-      id: {
-        type: DataTypes.INTEGER,
-        autoIncrement: true,
-        primaryKey: true,
-      },
-      gitHubId: {
-        type: DataTypes.INTEGER,
-        allowNull: true,
-      },
-      patreonId: {
-        type: DataTypes.INTEGER,
-        allowNull: true,
-      },
-      authKey: {
-        type: DataTypes.CHAR(32),
-        allowNull: false,
-        unique: true,
-      },
-      data: {
-        type: DataTypes.TEXT('long'),
-        allowNull: false,
-      },
-      createdAt: {
-        type: DataTypes.DATE, // this is actually DATETIME
-        defaultValue: DataTypes.NOW,
-        allowNull: false,
-      },
-      lastSeenAt: {
-        type: DataTypes.DATE, // this is actually DATETIME
-        defaultValue: DataTypes.NOW,
-        allowNull: false,
-      },
-    }).then(result => Promise.all([
-      queryInterface.addIndex('User', ['gitHubId']),
-      queryInterface.addIndex('User', ['patreonId']),
-    ]));
+    return queryInterface
+      .createTable("User", {
+        id: {
+          type: DataTypes.INTEGER,
+          autoIncrement: true,
+          primaryKey: true,
+        },
+        gitHubId: {
+          type: DataTypes.INTEGER,
+          allowNull: true,
+        },
+        patreonId: {
+          type: DataTypes.INTEGER,
+          allowNull: true,
+        },
+        wclId: {
+          type: DataTypes.INTEGER,
+          allowNull: true,
+        },
+        authKey: {
+          type: DataTypes.CHAR(32),
+          allowNull: false,
+          unique: true,
+        },
+        data: {
+          type: DataTypes.TEXT("long"),
+          allowNull: false,
+        },
+        createdAt: {
+          type: DataTypes.DATE, // this is actually DATETIME
+          defaultValue: DataTypes.NOW,
+          allowNull: false,
+        },
+        lastSeenAt: {
+          type: DataTypes.DATE, // this is actually DATETIME
+          defaultValue: DataTypes.NOW,
+          allowNull: false,
+        },
+      })
+      .then((result) =>
+        Promise.all([
+          queryInterface.addIndex("User", ["gitHubId"]),
+          queryInterface.addIndex("User", ["patreonId"]),
+          queryInterface.addIndex("User", ["wclId"]),
+        ])
+      );
```

Comment:
this needs to be implemented as a separate, new migration. patching the old migration only works for newly spun-up databases. technically the wowa db could be dropped and rebuilt, but that will invalidate all existing users.